contracts/examples/hello-world/programs/hello-world/Cargo.toml

```diff
@@ -16,5 +16,5 @@ cpi = ["no-entrypoint"]
 default = []
 
 [dependencies]
-anchor-lang = "0.23.0"
-chainlink_solana = "0.1.2"
+anchor-lang = "0.25.0"
+chainlink_solana = "1.0.0"
```

Comment:
We didn't change versions for this update, right?

- So deployed contracts are 1.0.0, and this updated contracts will also be 1.0.0? 
- Will this mean we can fetch versions for old and new contracts and both will be the same?
- Why don't we bump this to 1.1.0?

Reply:
I'll update this to use `1.0`, we're not bumping the library to `1.1` because there wasn't any changes there though. The user facing queries are all stable and will work with both versions